src/docs/truffle/reference/truffle-commands.md

```diff
@@ -77,15 +78,13 @@ Optional parameters:
 Helper to create new contracts, migrations and tests.
 
 ```shell
-truffle create (contract|migration|test) <ArtifactName>
+truffle create <artifact_type> <ArtifactName>
```

Comment:
Ooh this exposes a bad naming choice. We should stop using `artifact` here, so we can keep that word for specifically the JSON files in `build/contracts/`

Reply:
Without taking too much time to think of ideas, maybe we can just call it `object`? Or `entity`?

Reply:
Just an idea, what if we let all these things fall under the name of artifact and we differentiate them by prefacing them with the type.  Like, 'migration artifact' or `compiled contract artifact`?  Is that too much?

Reply:
Or what about 'abstraction'?

Reply:
prefixing `artifact` works. it's tough because those two words (artifact/abstraction) are both used to refer to very specific things, and they're already confusing terms. So anything that decreases risk of confusion is good 👍